safe-cli/subcommands/helpers.rs

```diff
@@ -28,22 +28,27 @@ pub fn get_from_arg_or_stdin(
     message: Option<&str>,
 ) -> Result<String, String> {
     match target_arg {
-        Some(ref t) if t == "" => get_from_stdin(message),
+        Some(ref t) if t.is_empty() => {
+            let val = get_from_stdin(message)?;
+            Ok(String::from_utf8_lossy(&val).to_string())
```

Comment:
I worry about using from_utf8_lossy(), as it silently alters the user's input if any non utf8 chars encountered.  User may be using a locale other than UTF-8, or paste in a URL with invalid chars obtained from elsewhere.    I would think its better to use the non lossy version and throw an error.